src/app/database.py

```diff
@@ -122,6 +122,8 @@ def get_schema_metadata(survey_id, version) -> SchemaMetadata:
 
     for schema in schemas_result:
         return_metadata = schema.to_dict()
+        if len(return_metadata) == 0:
+            return None
```

Comment:
I don't like the idea of returning `None` here, this should probably pass the issue straight to the exception filter

Reply:
Removed this logic